skeleton/techdoc/docs/rhoai.md

```diff
@@ -12,16 +12,27 @@ This document will outline how you can run build and run your sample application
 
 ## Running the Sample
 
-1) Navigate to the OpenShift AI workbench created for your sample application
+1) While on Console, navigate to the `OpenShift AI`
```

Comment:
Images won't show when displayed as techdoc. They're fine to include since they'll show in the GitHub repo that's generated, but we probably want to be a bit more descriptive here about where to click (that icon with 9 squares, not sure what it's called?)